Dockerfile-localnet

```diff
@@ -17,7 +17,8 @@ COPY go.mod .
 COPY go.sum .
 RUN go mod download
 COPY version.sh .
-COPY --exclude=*.sh --exclude=*.md --exclude=*.yml . .
+#COPY --exclude=*.sh --exclude=*.md --exclude=*.yml . .
+COPY . .
```

Comment:
Is it a change for debugging purposes that you forgot to revert? Seems like not necessary to me

Reply:
reverted